src/syrupy/utils.py

```diff
@@ -1,9 +1,37 @@
 import os
-from typing import Generator
+from typing import (
+    Any,
+    Generator,
+    List,
+    Optional,
+    Set,
+)
 
 from .constants import SNAPSHOT_DIRNAME
 
 
+def get_targeted_snapshots_from_targeted_files(
+    targeted_files: Optional[Set[Any]],
+) -> List[str]:
+    if not targeted_files:
+        return []
+
+    split_paths = [file_path.split("/") for file_path in targeted_files]
+
+    return [
+        os.path.abspath(
+            os.path.join(
+                *[
+                    *split_path[:-1],
+                    SNAPSHOT_DIRNAME,
+                    ".".join([os.path.splitext(split_path[-1])[0], "ambr"]),
```

Comment:
We shouldn't make assumptions around the file extension, especially as various syrupy extensions have different file extensions

Reply:
Given that you can pass in glob patterns, I'll swap over to using that to determine potential snapshots to look out for.